db/post_migrate/20250526063633_copy_add_groups_to_about_component_settings.rb

```diff
@@ -10,14 +10,15 @@ class CopyAddGroupsToAboutComponentSettings < ActiveRecord::Migration[7.2]
 
```

Comment:
Sorry but since you are touching the migration, can you also update it to avoid using `SiteSetting.set` and use raw SQL to update the site settings instead? Thank you!